pkg/cephfs/volume.go

```diff
@@ -17,27 +17,17 @@ limitations under the License.
 package cephfs
 
 import (
-	"fmt"
-	"os"
 	"path"
 
-	"k8s.io/klog"
+	"github.com/ceph/ceph-csi/pkg/util"
 )
 
 const (
 	cephVolumesRoot = "csi-volumes"
 
-	namespacePrefix = "ns-"
+	namespacePrefix = "fsvolumens-"
```

Comment:
@poornimag any specific reason for namespace prefix change here ?

Reply:
Its changed to the default value specified in the ceph subvol create code

Reply:
Thanks for clarifying

Reply:
@poornimag but Isnt it `fsvolumens_`

Reply:
Yes. This needs to 'fsvolumens_'

Reply:
Ideally, once user creation moves into the `ceph fs subvolume create ...` command I assume CSI code will not deal with this namespace name.

@ajarr Supporting user creation (based on some flag) is part of the implementation plan in the future?